build/cluster.tf

```diff
@@ -87,86 +93,94 @@ resource "google_container_cluster" "primary" {
       issue_client_certificate = false
     }
   }
-  enable_legacy_abac = "${lookup(var.cluster, "legacyAbac")}"
-  node_pool = [
-    {
-      name = "default"
-      node_count = "${lookup(var.cluster, "initialNodeCount")}"
-      node_config = {
-        machine_type = "${lookup(var.cluster, "machineType")}"
-        oauth_scopes = [
-          "https://www.googleapis.com/auth/devstorage.read_only",
-          "https://www.googleapis.com/auth/logging.write",
-          "https://www.googleapis.com/auth/monitoring",
-          "https://www.googleapis.com/auth/service.management.readonly",
-          "https://www.googleapis.com/auth/servicecontrol",
-          "https://www.googleapis.com/auth/trace.append",
-        ]
-
-        tags = ["game-server"]
-        timeouts = {
-          create = "30m"
-          update = "40m"
-        }
+
+  node_pool {
+    name       = "default"
+    node_count = "${var.cluster["initialNodeCount"]}"
+
+    node_config {
+      machine_type = "${var.cluster["machineType"]}"
+
+      oauth_scopes = [
+        "https://www.googleapis.com/auth/devstorage.read_only",
+        "https://www.googleapis.com/auth/logging.write",
+        "https://www.googleapis.com/auth/monitoring",
+        "https://www.googleapis.com/auth/service.management.readonly",
+        "https://www.googleapis.com/auth/servicecontrol",
+        "https://www.googleapis.com/auth/trace.append",
+      ]
+
+      tags = ["game-server"]
+    }
+  }
+
+  node_pool {
+    name       = "agones-system"
+    node_count = 1
+
+    node_config {
+      preemptible  = true
```

Comment:
Why are we putting the agones controller onto a preemptible vm?

Reply:
Nice catch, I think this is a left-over from other Terraform configuration example.